patches/print.fontforge.20201107.diff

```diff
@@ -0,0 +1,386 @@
++SIZE (fontforge-fontforge-20201107_GH0.tar.gz) = 19485522
+diff --git a/print/fontforge/files/patch-fontforgeexe_startui.c b/print/fontforge/files/patch-fontforgeexe_startui.c
+deleted file mode 100644
+index a7e23b7834a6..000000000000
+--- a/print/fontforge/files/patch-fontforgeexe_startui.c
++++ /dev/null
+@@ -1,25 +0,0 @@
+---- fontforgeexe/startui.c.orig	2019-08-01 08:28:36 UTC
+-+++ fontforgeexe/startui.c
+-@@ -1099,10 +1099,6 @@ int fontforge_main( int argc, char **argv ) {
+-     CheckIsScript(argc,argv); /* Will run the script and exit if it is a script */
+- 					/* If there is no UI, there is always a script */
+- 			                /*  and we will never return from the above */
+--#ifdef FONTFORGE_CAN_USE_GDK
+--    gdk_init(&argc, &argv);
+--    gdk_set_allowed_backends("win32,quartz,x11");
+--#endif
+-     if ( load_prefs==NULL ||
+- 	    (strcasecmp(load_prefs,"Always")!=0 &&	/* Already loaded */
+- 	     strcasecmp(load_prefs,"Never")!=0 ))
+-@@ -1185,7 +1181,10 @@ int fontforge_main( int argc, char **argv ) {
+- 	}
+- #endif
+-     }
+--
+-+#ifdef FONTFORGE_CAN_USE_GDK
+-+    gdk_init(&argc, &argv);
+-+    gdk_set_allowed_backends("win32,quartz,x11");
+-+#endif
+-     ensureDotFontForgeIsSetup();
+- #if defined(__MINGW32__) && !defined(_NO_LIBCAIRO)
+-     //Load any custom fonts for the user interface
+diff --git a/print/fontforge/files/patch-m4_fontforge__arg__enable.m4 b/print/fontforge/files/patch-m4_fontforge__arg__enable.m4
+deleted file mode 100644
+index b9eba9a9f393..000000000000
+--- a/print/fontforge/files/patch-m4_fontforge__arg__enable.m4
++++ /dev/null
+@@ -1,11 +0,0 @@
+---- m4/fontforge_arg_enable.m4.orig	2019-08-01 08:28:36 UTC
+-+++ m4/fontforge_arg_enable.m4
+-@@ -232,7 +232,7 @@ AC_DEFUN([FONTFORGE_ARG_ENABLE_WOFF2],
+- AC_ARG_ENABLE([woff2],
+-         [AS_HELP_STRING([--enable-woff2],
+-                 [Enable WOFF2 support.])],
+--        [use_woff2=yes])
+-+        [use_woff2="${enableval}"])
+- if test x$use_woff2 = xyes ; then
+-     PKG_CHECK_MODULES([WOFF2],[libwoff2enc,libwoff2dec],
+-     [
+diff --git a/print/fontforge/files/patch-mk_layout.am b/print/fontforge/files/patch-mk_layout.am
+deleted file mode 100644
+index 6a55820f684c..000000000000
+--- a/print/fontforge/files/patch-mk_layout.am
++++ /dev/null
+@@ -1,11 +0,0 @@
+---- mk/layout.am.orig	2014-11-27 00:40:08 UTC
+-+++ mk/layout.am
+-@@ -52,7 +52,7 @@ HTDOCS_SUBDIR =
+- htdocsdir = ${docdir}$(HTDOCS_SUBDIR)
+- 
+- # Where pkg-config files go.
+--pkgconfigdir = ${libdir}/pkgconfig
+-+pkgconfigdir = ${PREFIX}/libdata/pkgconfig
+- 
+- # Where contributed Python scripts go.
+- pkgpythondatadir = $(pkgdatadir)/python
+diff --git a/print/fontforge/pkg-descr b/print/fontforge/pkg-descr
+index 5286247bb570..9fd86d3c2125 100644
+--- a/print/fontforge/pkg-descr
++++ b/print/fontforge/pkg-descr
+@@ -12,4 +12,4 @@ from imported bitmap images.
+ 
+ For more information, see
+ 
+-WWW: http://fontforge.sourceforge.net/
++WWW: https://fontforge.org/en-US/
```

Comment:
I think we can just use https://fontforge.org

Reply:
Well, it gets redirected anyways ;)

Reply:
Yes but it looks more formal. :-)

Reply:
Okay I guess you have a point, will make the switch ;)